src/main/kotlin/org/jitsi/jicofo/bridge/ExternalBridgeSelectionStrategy.kt

```diff
@@ -0,0 +1,110 @@
+package org.jitsi.jicofo.bridge
+
+import org.jitsi.utils.logging2.Logger
+import org.jitsi.utils.logging2.LoggerImpl
+import org.json.simple.JSONObject
+import org.json.simple.JSONValue
+import java.net.URI
+import java.net.http.HttpClient
+import java.net.http.HttpRequest
+import java.net.http.HttpResponse
+
+@Suppress("unused")
+class ExternalBridgeSelectionStrategy() : BridgeSelectionStrategy() {
+    private val httpClient: HttpClient = HttpClient
+        .newBuilder()
+        .connectTimeout(ExternalBridgeSelectionStrategyConfig.config.timeout)
```

Comment:
You can import `ExternalBridgeSelectionStrategyConfig.config` to make these calls shorter